tron/config/config_parse.py

```diff
@@ -161,6 +162,41 @@ def valid_node_name(value, config_context):
     return value
 
 
+def valid_master_address(value, config_context):
+    """Validates and normalizes Mesos master address.
+
+    Must be HTTP or not include a scheme, and only include
+    a host, without any path components.
+    """
+    valid_string(value, config_context)
+
+    # Parse with HTTP as default, only HTTP allowed.
+    scheme, netloc, path, params, query, fragment = urlparse(value, 'http')
+    if scheme != 'http':
+        msg = f"Only HTTP supported for Mesos master address, got {value}"
+        raise ConfigError(msg)
+
+    if params or query or fragment:
+        msg = f"Mesos master address may not contain path components, got {value}"
+        raise ConfigError(msg)
+
+    # Only one of netloc or path allowed, and no / except trailing ones.
+    # netloc is empty if there's no scheme, then we try the path.
+    path = path.rstrip('/')
+    if (netloc and path) or '/' in path:
+        msg = f"Mesos master address may not contain path components, got {value}"
```

Comment:
nit: you might want to make the msg a little different from the previous one to make debugging easier.

Reply:
The solution is the same (take everything out except the host address) so I think it's fine. I think they're basically the same problem.